src/gui/folderwatcher_linux.cpp

```diff
@@ -58,7 +58,7 @@ bool FolderWatcherPrivate::findFoldersBelow( const QDir& dir, QStringList& fullL
     } else {
         QStringList nameFilter;
         nameFilter << QLatin1String("*");
-        QDir::Filters filter = QDir::Dirs | QDir::NoDotAndDotDot|QDir::NoSymLinks;
+        QDir::Filters filter = QDir::Dirs | QDir::NoDotAndDotDot | QDir::NoSymLinks | QDir::Hidden;
         const QStringList pathes = dir.entryList(nameFilter, filter);
```

Comment:
Yes, but the current pattern here is to generate a full list and then call pathIsIgnored() on each entry afterwards. If we wanted to apply the "is hidden" part of the filter early, we'd need to make the "ignoreHidden" flag available which is currently an implementation detail of pathIsIgnored().

Instead I'd prefer to skip building that list altogether. Agreed?